extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -201,6 +201,46 @@ In the following example, we display a `toolbar` if the window width is less tha
 See live demos at [AMP By Example](https://ampbyexample.com/components/amp-sidebar/).
 {% endcall %}
 
+## Sidebar for Stories
+The sidebar is also usable within the `amp-story` [component](https://www.ampproject.org/stories/). However there are certain restrictions on what attributes and features are allowed. The following are **disallowed** attributes and features of sidebar within an `amp-story`:
```

Comment:
"However there are certain restrictions on what attributes and features are allowed. "

Is there a brief explanation on why some attributes are allowed and disallowed?

Reply:
I believe that some like side & open are disallowed because of default UI and behaviors we want to enforce for the sidebar. I believe that toolbar related attributes are due to the fact that we don't want to enable elements within the amp-sidebar to be displayed on other parts of the body. 

Can you confirm @gmajoulet or @newmuis ?

Do you think that it'd be worthwhile to give the explanation in the docs?

Reply:
I would move this section under the behavior section and just add a line that explains this is related to a consistent UI experience across the story platform.

Reply:
Done!